test/extended/builds/run_policy.go

```diff
@@ -0,0 +1,241 @@
+package builds
+
+import (
+	"fmt"
+	"strings"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+
+	buildapi "github.com/openshift/origin/pkg/build/api"
+	buildclient "github.com/openshift/origin/pkg/build/client"
+	buildutil "github.com/openshift/origin/pkg/build/util"
+	exutil "github.com/openshift/origin/test/extended/util"
+	kapi "k8s.io/kubernetes/pkg/api"
+)
+
+var _ = g.Describe("[builds][Slow] using build configuration runPolicy", func() {
```

Comment:
I'd like at least one test of this to be "fast" and in conformance suite that verifies the behavior - a single test that creates the build, slates some very fast ones that don't push, and then generates multiples.  Basically, like deployment, this can and will flake, and I want a ~15-20s test in conformance that will generate flakes so we have to fix them.